Assets/Scripts/Entity/EntityCommandGiver.cs

```diff
@@ -22,7 +22,13 @@ private void Update()
 
         Ray ray = mainCamera.ScreenPointToRay(Mouse.current.position.ReadValue());
 
-        if (!Physics.Raycast(ray, out RaycastHit hit, Mathf.Infinity, layerMask)) { return; }
+        if (!Physics.Raycast(ray, out RaycastHit hit, Mathf.Infinity, layerMask)) { return; } // No hit
+
+        if (hit.collider.TryGetComponent(out Entity entity) && !entity.isOwned)   // hit an enemy entity
+        {
+            TryAttack(entity);
+            return;
+        }
```

Comment:
Consider allowing selected armies to chase targeted armies until in firing range

&nbsp;

GameDev.tv UnitMovement.cs:
```
[ServerCallback]
private void Update()
{
    Targetable target = targeter.GetTarget();

    if (target != null)
    {
        if ((target.transform.position - transform.position).sqrMagnitude > chaseRange * chaseRange)
        {
            agent.SetDestination(target.transform.position);
        }
        else if (agent.hasPath)
        {
            agent.ResetPath();
        }

        return;
    }

    if (!agent.hasPath) { return; }

    if (agent.remainingDistance > agent.stoppingDistance) { return; }

    agent.ResetPath();
}
```